### PR TITLE
fix some broken contents links

### PR DESCRIPTION
**Pull request by pengqinglan**
**- What I did**
fix some broken links on contents
**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: Qinglan Peng <qinglanpeng@zju.edu.cn>

### CODE REVIEW

File: docs/reference/run.md [hunk 1]
```diff
@@ -63,15 +63,15 @@ Only the operator (the person executing `docker run`) can set the
...
  - [Detached vs foreground](#detached-vs-foreground)
-     - [Detached (-d)](#detached-d)
+     - [Detached (-d)](#detached--d)
      - [Foreground](#foreground)
  - [Container identification](#container-identification)
-     - [Name (--name)](#name-name)
+     - [Name (--name)](#name---name)
      - [PID equivalent](#pid-equivalent)
- - [IPC settings (--ipc)](#ipc-settings-ipc)
- - [Network settings](#network-settings)
```
Comment (AkihiroSuda) [inline docs/reference/run.md]: this line should be kept

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

**Comment by AkihiroSuda** [comment]
Can we detect such broken links in CI?

File: docs/reference/run.md [hunk 2]
```diff
@@ -63,15 +63,15 @@ Only the operator (the person executing `docker run`) can set the
...
+ - [IPC settings (--ipc)](#ipc-settings---ipc)
+ - [Network settings](#network---settings)
  - [Restart policies (--restart)](#restart-policies-restart)
```
Comment (AkihiroSuda) [inline docs/reference/run.md]: Please fix this line as well?

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: docs/reference/run.md (hunk 1 above)
Comment (pengqinglan) [inline docs/reference/run.md]: Yeah, it should be kept, I corrected the wrong line

**Comment by pengqinglan** [comment]
These links point to the current page, I'm afraid that can't be detected in CI

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Comment by pengqinglan** [comment]
Thank you!


Original Message
Sender:Akihiro Sudanotifications@github.com
Recipient:docker/dockerdocker@noreply.github.com
Cc:Qinglan Pengqinglanpeng@zju.edu.cn; Authorauthor@noreply.github.com
Date:Monday, Dec 12, 2016 16:12
Subject:Re: [docker/docker] fix some broken contents links (#29308)


@AkihiroSuda approved this pull request.
LGTM

**Review by MHBauer — APPROVED** [review]
LGTM

links work after changes.

**Comment by thaJeztah** [comment]
@pengqinglan can you squash your commits, so that there's only a single commit in this PR? You can find information on how to squash here; https://docs.docker.com/opensource/workflow/work-issue/#/pull-and-rebase-frequently

**Comment by pengqinglan** [comment]
@thaJeztah I have squashed the two commits into a single commit, thanks for your instruction！

**Review by thaJeztah — APPROVED** [review]
perfect, thanks!

LGTM

**Pull request merged**